src/index.js

```diff
@@ -11,6 +12,7 @@ app.use(cors())
 app.use(express.json())
 app.use(categoryRouter)
 app.use(expensesRouter)
+app.use(usersRouter)
```

Comment:
It grows up and with more endpoints there is going to grow even further. How about gather all routers into an array in common file and import them here an this array and make forEach on all of them?